tutorial1/step2a.md

```diff
@@ -15,8 +15,8 @@ Congratulations you've just run your first command on command line!
  A **directory** is just another name for a folder\*. When you are in a 
  command line environment you are always *somewhere* in the filesystem and 
  this is known as your **working directory**.  This can be a weird concept 
- if you only haven't used command line before but you will get used to 
- this really quickly - don't worry about it.
+ if you have only used command line before, but you will get used to it
```

Comment:
haven't or have not, I'm fine with the rest

Reply:
Can you confirm is resolved by b9df008?

Reply:
Yes this is resolved